### PR TITLE
New method of running

### PR DESCRIPTION
**Pull request by bdw429s**
OK, I have done an experiment to try and shave off about 4 seconds or so of startup time.  Results may vary based on CPU power, etc.  As much as I love the task runners and print helper, etc they rely on a bunch of modules that load up when CommandBox starts.  I've changed the task component to a regular .cfm file that is directly invoked, bypassing the normal CommandBox bootstrap.  I've also updated the readme to show how to call it now.  

```
box -clishellpath=/path/to/TestRunner.cfm
```
All of the rest of the conventions as far as the test files and such should work the same.  I just had to add a bit of boilerplate code to it to do what I needed.  Please give this a try and see if it improves the execution time. I do think you'll need the absolute path to the testrunner.cfm file.

### CODE REVIEW

File: TestRunner.cfm [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+				}
+			}
+		)
+		.getMemento( includeDebugBuffer=true );
+
+	new CodewarsReporter().render( testData );
+
+	// Set exit code to 1 on failure
+	if ( testData.totalFail || testData.totalError ) {
+		return 1;
```
Comment (kazk) [inline TestRunner.cfm]: Does this set the exit code properly? It doesn't seem like it does when I try.

**Review by kazk — COMMENTED** [review]
Thanks @bdw429s! I don't have numbers yet, but it seems much faster.
The only issue is that the exit code on failure/error doesn't seem to be set anymore.

File: TestRunner.cfm (hunk 1 above)
Comment (bdw429s) [reply TestRunner.cfm]: Oh good call, that was another thing the CommandBox bootstrap provided.  Easy fix, let me add that in.

Comment (bdw429s) [reply TestRunner.cfm]: @kazk Ok, updated (pushed new commit to branch/pull).  Please give that another try.

**Review by kazk — APPROVED** [review]

**Comment by kazk** [comment]
3-4s speed up on my laptop :rocket:
I'll let you know when this is deployed.

**Comment by bdw429s** [comment]
Sweet, sounds good.

**Comment by kazk** [comment]
Deployed

**Pull request merged**